runtime/src/main/java/org/corfudb/runtime/view/SealLayoutServers.java

```diff
@@ -0,0 +1,59 @@
+package org.corfudb.runtime.view;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.BaseClient;
+import org.corfudb.runtime.exceptions.QuorumUnreachableException;
+
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Created by zlokhandwala on 3/7/17.
+ */
+public class SealLayoutServers {
+
+    public CompletableFuture<Boolean> asyncSealLayoutServers(CorfuRuntime runtime, Layout layout, long sealEpoch) {
+
+        CompletableFuture<Boolean> completableLayoutServersSeal = new CompletableFuture<>();
+
+        CompletableFuture.supplyAsync(() -> {
```

Comment:
By the way, this work is queued on a common thread pool, I think we need a more controllable way to spawn this work.